### ISSUE TITLE
Minimum audio volume is too loud with bluetooth headset

### ISSUE THREAD

**Issue by vladtaranenkoSA** [RenderHeads/UnityPlugin-AVProVideo#1487]
When playing HLS video with bluetooth headset connected, minimum volume is loud, maximum is VERY loud.

**Your Setup (please complete the following information):**
 - Unity version: 2019.4.23f1
 - AVPro Video version (number and edition (trial/core/ultra/enterprise)): 2.6.7
 - Operating system version: iOS 15.6
 - Device model: IPhone XR, headset Beats Flex
 - Video specs (resolution, frame-rate, codec, file size): HLS

**To Reproduce**
1. Connect BT Headset
2. Play HLS Video

**Screenshots**
![image](https://user-images.githubusercontent.com/92296634/224984081-86ca8350-e084-4281-a3ee-84a6afc3b339.png)

**Comment by Chris-RH** [comment]
Any more info @vladtaranenkoSA ?

**Comment by Chris-RH** [comment]
Closed due to lack of reply

**Comment by vladtaranenkoSA** [comment]
Hello, sorry for late reply.
So regarding your questions

Updated to latest AVPro version 2.7.3
Issue still reproduces in our app, video has loud sound in headset when on min volume, about 50% of normal volume, 100% volume is really loud. When playing sound from another audio source in our app, volume output is normal, and I'm able to mute it. 

Possible workaround
Using Headphone accomodations OR forgetting and pairing headset again fixes this issue, but it comes back after a while. I can't say for sure, tried rebooting phone, uninstalling/reinstalling app, issue wont come back immediately. Today I built app with updated AVPro Video and got this issue again. 
Headphone accomodations is only available for Beats and Apple headsets, and we got this issue on other headset as well.

1. How is it playing out of the device speakers rather than the bluetooth speakers?
Speakers play normally, able to mute it with min volume.

2. Have you tested playback using a new project and our demo mediaplayer scene (first with the default video, and then with your HLS)? 
Could not reproduce it in demo with latest AVPro version 2.7.3 on iOS. Tested right after got this issue again in our app. Demo min volume was muted, and our app was loud.

3. Have you tested different media?
Tried both demo resources and our own HSL streams

4. Does this occur using other devices?
Iphone 8S with JBL Reflect mini headset also got this issue

5. Does this occur using other apps on the same device?
Other apps behave normally, checked on Safari, Youtube, Tiktok, random unity game from store.
